media/gpu/v4l2/v4l2_device.cc

```diff
@@ -95,6 +95,9 @@ uint32_t V4L2Device::VideoPixelFormatToV4L2PixFmt(VideoPixelFormat format) {
 // static
 uint32_t V4L2Device::VideoCodecProfileToV4L2PixFmt(VideoCodecProfile profile,
                                                    bool slice_based) {
+#if BUILDFLAG(USE_LINUX_V4L2)
+  slice_based = false;
+#endif
```

Comment:
also fixed these line.
Does it make sense?

Reply:
+1